http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonPayloadIT.java

```diff
@@ -0,0 +1,184 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Date;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class JsonPayloadIT {
+
+    @QuarkusApplication(classes = { FootballTeamResource.class, FootballTeam.class
+    }, properties = "oidcdisable.properties")
+    static RestService app = new RestService();
+
+    @Test
+    @Order(1)
+    public void sendSuccessfulJSONPayloadFromAFile() throws IOException {
+
+        String jsonPayload = FileUtils.readFileToString(
```

Comment:
Wouldn't be better to have one method for this as it's used multiple times and most of the values are same except the file?

Reply:
Yes, I'll refactor it and create a method to improve code readability, thanks.